tracee-rules/signature.go

```diff
@@ -97,7 +98,7 @@ func findRegoSigs(dir string) ([]types.Signature, error) {
 		}
 		sig, err := regosig.NewRegoSignature(string(regoCode), regoHelpersCode)
 		if err != nil {
-			log.Printf("error creating rego signature with: %s: %v ", regoCode[0:20], err)
+			log.Printf("error creating rego signature with: %s: %v ", regoCode[0:bytes.Index(regoCode, []byte("\n"))], err)
```

Comment:
From the GoDOC for bytes.Index:

`Index returns the index of the first instance of sep in s, or -1 if sep is not present in s.`

So this can potentially cause a panic. Can we not do `regoCode[0:len(regoCode)]` or just `regoCode` ?

Reply:
That will print the entire rego code, which is a bit much in my opinion.
I'll add a check to see if the returned value is -1.
Good catch - thanks!